### PR TITLE
Librarize

### PR DESCRIPTION
**Pull request by StrongestNumber9**
Refactors whole project to be a library instead of standalone flooder. Example clientside use can be found [here](https://github.com/StrongestNumber9/rlp_10/blob/dev/src/main/java/com/teragrep/rlp_10/Main.java)

Has basically zero failure tolerance as it is not meant to be anything except "run until stopped or something breaks"

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -117,44 +110,80 @@
...
+              </execution>
+            </executions>
+          </plugin>
+        </plugins>
+      </build>
+      <distributionManagement>
+        <repository>
+          <id>github</id>
+          <name>GitHub Packages</name>
+          <url>https://maven.pkg.github.com/${env.GITHUB_REPOSITORY}</url>
```
Comment (kortemik) [inline pom.xml:184]: is this intentional ?

**Comment by kortemik** [comment]
i think elegant objects principles could be easily implemented regarding naming of methods

File: pom.xml (hunk 1 above)
Comment (StrongestNumber9) [reply pom.xml:184]: It is only used on deploy and it autofills it to teragrep/rlp_09 as expected

**Review by eemhu — COMMENTED** [review]
looks good to me

**Review by kortemik — APPROVED** [review]

**Pull request merged**